src/Ombi/ClientApp/src/app/discover/components/carousel-list/carousel-list.component.ts

```diff
@@ -224,7 +226,11 @@ export class CarouselListComponent implements OnInit {
                 this.movies = await this.searchService.popularMoviesByPage(this.currentlyLoaded, this.amountToLoad);
                 break;
             case DiscoverType.Trending:
+              if(this.isNewTrendingSourceEnabled) {
```

Comment:
it looks like we are doing the same thing but two different ways.

For Tv shows it's done in the TvEngine, but for Movies it's done in the UI layer? 

Can we move the Movie implementation to the backend?

Reply:
Yeah I initially wanted to build new endpoints for both but when I got to TV I had no choice but to 'break' the existing endpoint.
It didn't feel silly at that time but now that you point it out, I'll make it more consistent.